plugins/crypto/mbedtls/certificategroup.c

```diff
@@ -444,23 +455,31 @@ mbedtlsCheckSignature(const mbedtls_x509_crt *cert, mbedtls_x509_crt *issuer) {
                                   hash, hash_len, sig->p, sig->len) == 0);
 }
 
+typedef enum {
```

Comment:
This could be a bitfield.
So an enum with 1,2,4 values.
Where several verification steps are done at once.